src/views/Accessibility/AccessibilityRequestDetailPage.test.tsx

```diff
@@ -440,24 +440,28 @@ describe('AccessibilityRequestDetailPage', () => {
     describe('Add a note', () => {
       it('shows a success message', async () => {
         const createNoteMocks = [
-          defaultQuery,
+          withNotesQuery,
           {
             request: {
               query: CreateAccessibilityRequestNote,
               variables: {
-                requestID: 'a11yRequest123',
-                note: 'I am an important note',
-                shouldSendEmail: false
+                input: {
+                  requestID: 'a11yRequest123',
+                  note: 'This is very well done',
+                  shouldSendEmail: false
+                }
               }
             },
             result: {
               data: {
                 createAccessibilityRequestNote: {
-                  id: 'noteID2',
-                  createdAt: 'time',
+                  id: 'noteID1',
+                  createdAt: new Date().toISOString(),
                   authorName: 'Common Human',
-                  note: 'This is such a success'
-                }
+                  note: 'This is very well done'
+                },
+                userErrors: null,
+                __typename: 'CreateAccessibilityRequestNotePayload'
```

Comment:
is this necessary? I thought `addTypename={false}` removed the need to have this.

Reply:
true.  that was me grasping at straws getting the test to pass.